recipe/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "tsai" %}
+{% set version = "0.3.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: d54d88ca40d27d29e96adc98cb9641c848411caedaf517a93dce3b64c4d37915
+
+build:
+  number: 0
+  skip: True  #[py<37 or py>310 or s390x or ppc64le]
+  entry_points:
+    - nb2py=tsai.export:nb2py
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - wheel
+    - setuptools
+  run:
+    - fastai >=2.7.12
+    - imbalanced-learn >=0.8.0
+    - psutil >=5.4.8
+    - python
+    - pyts >=0.12.0
+    - pytorch >=1.7,<2.1
+    - ipykernel
```

Comment:
`ipykernel >6` is needed only for development https://github.com/timeseriesAI/tsai/blob/98d5adfaa7f4814dea293902070039e2e859ba10/tsai/export.py#L82

Reply:
I have removed it from run, it is still implicit dependency but not needed in host